src/Themes/NexusMods.Themes.NexusFluentDark/Styles/UserControls/Sorting/SortingSelectionStyles.axaml

```diff
@@ -0,0 +1,20 @@
+<Styles xmlns="https://github.com/avaloniaui"
```

Comment:
We should do without this if we have nothing to put in here by the end of this PR.
This might be assimilated into whatever parent view anyways later when we integrate into Loadout view etc